### PR TITLE
Remove redirect to blank page when token received

### PR DESCRIPTION
**Pull request by y0rrrsh**

### CODE REVIEW

File: VkAudio/viewcontroller/UserAudiosViewController.swift [hunk 1]
```diff
@@ -9,14 +9,14 @@
 import Foundation
 import UIKit
 
-class UserAudiosViewController: UIViewController{
+class UserAudiosViewController: UIViewController {
     
     internal static let STORYBOARD_ID: String = "controller_user_audios"
```
Comment (taviscaron) [inline VkAudio/viewcontroller/UserAudiosViewController.swift]: if a type of the declared variable can be taken from the context it's not needed to specify it explicitly;
`internal static let STORYBOARD_ID = "controller_user_audios"` works as well

File: VkAudio/viewcontroller/UserAudiosViewController.swift [hunk 2]
```diff
@@ -9,14 +9,14 @@
...
+    internal static let SEGUE_ID: String = "login_to_user_audios"
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        let audios: String = VkSDK.instance!.requestManager!.getAudios("124259152")
-        print(audios)
+        VkSDK.instance?.getAudios{result in print(result)}
```
Comment (taviscaron) [inline VkAudio/viewcontroller/UserAudiosViewController.swift]: error handling ?

Comment (y0rrrsh) [inline VkAudio/viewcontroller/UserAudiosViewController.swift]: working on it

File: VkAudio/vk/RequestManager.swift [hunk 3]
```diff
@@ -19,10 +19,26 @@ class RequestManager {
...
-        return requestUrl
+        Alamofire.request(.GET, URL_REQUEST_BASE + apiMethod, parameters: params)
+            .responseJSON {
+                response in
+                let result = response.result
+                if result.isSuccess {
+                    onResult(result: result.value!)
+                }
+                else if onError != nil {
+                    onError!(error: result.error!)
```
Comment (taviscaron) [inline VkAudio/vk/RequestManager.swift]: `onError` is declared as optional type so here it can cause crash

**Pull request merged**